pkg/gce-pd-csi-driver/utils.go

```diff
@@ -56,14 +57,30 @@ func NewNodeServiceCapability(cap csi.NodeServiceCapability_RPC_Type) *csi.NodeS
 	}
 }
 
+// Reflect magic below simply clears Secrets map from request.
```

Comment:
It would be good to document all the assumptions about how secret fields are named in the csi spec.